frontend/src/components/Typography/Head1/index.module.css

```diff
@@ -0,0 +1,6 @@
+.head1 {
```

Comment:
Would you mind keeping `heading1` to be consitent with design naming?

Reply:
Makes sense, changed in this as well as in the other heading components.